### DISCUSSION TITLE
Precisely keep towny plot if needed to update to 1.18 in the future.

### DISCUSSION THREAD

**Discussion by MisterSick** [TownyAdvanced/Towny#5130] [Q&A] [answered]
Hi guys 
Because of lacking English ability...
Couldnt find the answer how to precisely keep towny plot if I needed to update spigot or paper version in the future. (like 1.18 cave part 2)
Only found a tool like MCA selector but it wasnt design for any claim plugin.
Any suggest?
Thanks for the people who answers.
A lot of appreciate.

**Comment by TheFlagCourier** [comment, marked as answer]
You shouldn't have to worry too much about this, as Towny iworks on a virtual grid of the X and Z axis.

 As long as the world remains the same, and that you don't change the Towny plot size, you can keep updating your world to newer versions and not see much of a difference. Just add the  `--forceUpgrade` to your startup script  when you transition to a new Minecraft release. Towny doesn't really care a whole lot about the y-axis, so you shouldn't see a problem going into 1.18+.
 
 That being said, you may want to wait for a Towny Release that _supports_ 1.18. This allows Towny to add the new blocks and go through some tests before you put your own world at risk. Otherwise, you better be running tests before you go to production.

The only problems you would typically need to worry about would be: if you are _deleting_ your world and generating a new one, or you changed your plot sizes. This is when you typically need to either restore from a backup, or delete your Towny data.

**Reply by TheFlagCourier** [reply]
PS: You will also want to wait for Spigot/Paper builds to mature. Usually around 2 weeks is enough.

**Reply by MisterSick** [reply]
Thank you for your respond Sir!
Yes, I do worry about regenerating wildness to get new feature by delete them.
(sorry not typed in the question about this)
Dont want to delete any plot accidentally but if do it on manual not by Towny data it might happen.
Backup is the only option right?
Thanks again for your reading :)
